test/unit/preloaded-vl-proza-message.test.html

```diff
@@ -0,0 +1,89 @@
+<!doctype html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <script src="../../node_modules/@webcomponents/webcomponentsjs/webcomponents-lite.js"></script>
+    <script src="../../node_modules/web-component-tester/browser.js"></script>
+    <script src="../../node_modules/sinon/pkg/sinon.js"></script>
+    <script src="../../node_modules/sinon-chai/lib/sinon-chai.js"></script>
+
+    <script type="module" src="../../vl-proza-message.src.js"></script>
+</head>
+
+<body>
+<test-fixture id="vl-proza-preloaded-messages-fixture">
+    <template>
+        <div>
+            <vl-proza-message-preloader id="preloader" data-vl-domain="foo"></vl-proza-message-preloader>
+            <vl-proza-message id="message-1" data-vl-domain="foo" data-vl-code="bar" data-vl-preload></vl-proza-message>
+            <vl-proza-message id="message-2" data-vl-domain="foo" data-vl-code="foo" data-vl-preload></vl-proza-message>
+        </div>
+    </template>
+</test-fixture>
+
+<script type="module">
+    import { VlProzaMessage, VlProzaMessagePreloader } from '../../vl-proza-message.src.js';
+    import fetchMock from 'fetch-mock/esm/client.mjs';
+
+    suite('preloaded vl-proza-message', () => {
```

Comment:
Conventie normaal is de tag van de webcomponent, in dit geval vl-proza-message-preloader.

Reply:
De preloader heeft zijn eigen suite. 
Dit is de suite bedoeld voor de combinatie van preloader en message. Daarom dat ik de conventie niet kon volgen. Een alternatieve naam is ook "vl-proza-message & vl-proza-message-preloader" natuurlijk. Voorkeuren?

Reply:
Nee geen voorkeur, dus prima zo.